src/app.js

```diff
@@ -16,6 +17,7 @@ const App = () => (
     <Home path="/">
       <Processors path="processors">
         <ProcessorsHome path="/" />
+        <DeviceInfo path="/:id" />
```

Comment:
Why `DeviceInfo`? Maybe call it `ProcessorInfo` and move to `/components/processors/processor-info.js`?

Not sure that motherboards, processors and videocards will have same params and layout